console.go

```diff
@@ -358,8 +359,10 @@ func consoleDefaultFormatCaller(noColor bool) Formatter {
 		if len(c) > 0 {
 			cwd, err := os.Getwd()
 			if err == nil {
-				prefix := cwd + "/"
-				c = strings.TrimPrefix(c, prefix)
+				rel, err := filepath.Rel(cwd, c)
+				if err == nil {
```

Comment:
Please make this a onliner `if rel, err := …`

Reply:
Resolved in 4f37b15.